Library/Formula/poppler.rb

```diff
@@ -17,26 +17,31 @@ class Poppler < Formula
   deprecated_option "with-qt4" => "with-qt"
   deprecated_option "with-lcms2" => "with-little-cms2"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'cairo'
-  depends_on 'fontconfig'
-  depends_on 'freetype'
-  depends_on 'gettext'
-  depends_on 'glib'
-  depends_on 'gobject-introspection'
-  depends_on 'jpeg'
-  depends_on 'libpng'
-  depends_on 'libtiff'
-  depends_on 'openjpeg'
+  if build.head?
+    depends_on :automake => :build
```

Comment:
I'm surprised it didn't. I could have sworn there was some mention of ` :autoconf ` and similar being deprecated somewhere. Maybe there's not an audit check, or I'm just wrong - The latter happens a lot. CC @mikemcquaid for detail on whether I'm being stupid or not :smile_cat: